release notes/v0.40.0.md

```diff
@@ -0,0 +1,249 @@
+k6 v0.40.0 is here! :tada: 
+#TODO 
+[Roadmap and future plans section](#roadmap-and-future-plans).
+
+## New Features and notable changes!
+
+### Breaking change!: Main module/script no longer taints the global scope [#2571](https://github.com/grafana/k6/pull/2571)
```

Comment:
```suggestion
### [Breaking change] Main module/script no longer taints the global scope [#2571](https://github.com/grafana/k6/pull/2571)
```
Or maybe can we use a :warning: emoji?

Reply:
That might not be a bad idea :thinking:

Reply:
like now?

Reply:
![2022-08-31-160631](https://user-images.githubusercontent.com/2103732/187698558-4a4ffe16-a369-4395-81dc-00c091cf89a2.png)
It seems a bit too much, maybe could it be just one per item? I think also you have to use `:warning:` for getting it rendered correctly.

Reply:
[b358c6a](https://github.com/grafana/k6/pull/2669/commits/b358c6aa17f7f9e5eb613996f5516113422517ac)